ts/parts/Point.ts

```diff
@@ -609,7 +612,9 @@ class Point {
         // For higher dimension series types. For instance, for ranges, point.y
         // is mapped to point.low.
         if (pointValKey) {
-            point.y = (point as any)[pointValKey];
+            point.y = pointValKey.indexOf('custom.') === 0 ?
+                getPropertyValue(pointValKey, point) :
+                (point as any)[pointValKey];
```

Comment:
Can we move all repeated code into a separate method? Like this:
```js
getValueFor = function (point, key) {
  return key.indexOf('custom.') === 0 ? getPropertyValue(key, point) : (point as any)[key];
};
```

So the line 615 above would be simple `point.y = getValueFor(pointValKey, point);` Otherwise, if we ever will need to use that method again there's a high chance that `something.indexOf('custom.')` will be forgotten.

Reply:
The condition is more for performance reasons, to skip a function call. But there is another thing, why I make a difference. `getPropertyValue` should only work on `point.options` (so that is the mistake in the code snippet).

Reply:
> The condition is more for performance reasons, to skip a function call

Yes, I understand that, but I still believe in this case it's better to make a function call instead of code repetition.

> `getPropertyValue` should only work on point.options (so that is the mistake in the code snippet)

The snippet was created from the lines above comment:

```js
            point.y = pointValKey.indexOf('custom.') === 0 ?
                getPropertyValue(pointValKey, point) :
                (point as any)[pointValKey];
```
I noticed that sometimes you use `point` otherwise `point.options`. I thought it's intentional (Highcharts.Point instance vs user provided point options).